jsonschema-maven-plugin/java.header

```diff
@@ -0,0 +1,18 @@
+^/\*$
+^ \* Copyright (\d\d\d\d-)?\d\d\d\d VicTools\.$
+^ \*$
+^ \* Licensed under the Apache License, Version 2\.0 \(the \"License\"\);$
+^ \* you may not use this file except in compliance with the License\.$
+^ \* You may obtain a copy of the License at$
+^ \*$
+^ \*      http://www\.apache\.org/licenses/LICENSE-2\.0$
+^ \*$
+^ \* Unless required by applicable law or agreed to in writing, software$
+^ \* distributed under the License is distributed on an \"AS IS\" BASIS,$
+^ \* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\.$
+^ \* See the License for the specific language governing permissions and$
+^ \* limitations under the License\.$
+^ \*/$
+^$
+^package com\.github\.victools\.jsonschema\..*;$
+^$
```

Comment:
This file should not be necessary. If you build the whole multi-module project and not just the maven plugin, the single `java.header` file in the parent folder will be picked up.

Reply:
This will not be picked up when you build from within the child project. It will find the checkstyle.xml from the root, but the referenced java.header is apparently searched for in the directory where the build was started.

Reply:
Your line of thinking is clear. I will investigate a different dependency solution.

Reply:
That's what I meant above. You have to build everything, i.e. the whole maven multi-module project.

Reply:
> That's what I meant above. You have to build everything, i.e. the whole maven multi-module project.
This is not convenient right? Particularly when you are working on just one module you would like to minimize your build.

But if you prefer not to duplicate the java.header file I can live with that because it will only fail at the end of the build. I just have to surpress my unpleasant feeling when I get the checkstyle error thrown at me. :-)